src/rustdoc_gen.rs

```diff
@@ -97,12 +97,33 @@ impl<'a> CrateSource<'a> {
         all_crate_features.extend(self.regular_features());
         all_crate_features.into_iter().collect()
     }
+
+    fn features_not_on_blacklist(&self) -> Vec<String> {
```

Comment:
"Blacklist" as a term has a bit of a problematic history and unfortunate connotations, and also isn't sufficiently descriptive here since it doesn't specify how features may end up "blacklisted."

Try `heuristically_included_features()` or something to that effect, plus a generous doc comment on the function that describes how those features are selected?